### DISCUSSION TITLE
Strong

### DISCUSSION THREAD

**Discussion by Rozerinay** [nl-design-system/backlog#291] [Component suggestions]
<img width="480" alt="visuele weergave van een strong component" src="https://github.com/user-attachments/assets/e7a670ae-5eb1-49db-821e-683e3109b290">

We willen graag meer leren over de Strong component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook, Github of screenshots. Met deze informatie kunnen we bewijzen dat de component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1726054200544029)

---

## Naam
Strong

## Doel
Korte belangrijke tekst die vetgedrukt wordt weergegeven.

## Gerelateerde componenten
[Paragraph](https://github.com/orgs/nl-design-system/discussions/212), [Rich Text Content](https://github.com/orgs/nl-design-system/discussions/289)

## Onderzoek
[`<strong>`: The Strong Importance element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong)
Documentatie van Mozilla Developers Network over het `<strong>` HTML element.

[Screenreader support for text-level semantics](https://brucelawson.co.uk/2018/screenreader-support-for-text-level-semantics/)
Blogpost van Bruce Lawson waarin hij uitlegt dat de `<strong>` tag semantische waarde toevoegt voor visuele gebruikers, maar schermlezers deze nadruk meestal niet automatisch rapporteren, vanwege beperkte toegang tot de Accessibility Tree.

[The Difference Between `<b>` and `<Strong>` in HTML – Explained with Examples](https://www.freecodecamp.org/news/difference-between-b-and-strong-in-html/)
Blogpost waarin het verschil tussen  `<b>` en `<strong>` wordt uitgelegd.

**Comment by Rozerinay** [comment]
## Utrecht

[Storybook](https://nl-design-system.github.io/utrecht/storybook/?path=/story/css_css-emphasis--strong)

---

### Screenshots / Varianten

**Storybook**
<img width="400" alt="image" src="https://github.com/user-attachments/assets/9b78c37d-aa19-4889-8248-d195198c9b17">

---

### Notities
- Onder de naam: Emphasis. Variant heet: Strong.
- Wordt gestyled met de class `emphasis--strong`.

**Comment by Rozerinay** [comment]
## Amsterdam

---


### Screenshots / Varianten

**Publieke website**

<img width="500" alt="image" src="https://github.com/user-attachments/assets/bb5f15f8-09ad-430d-a262-d1823879311c">

<img width="300" alt="image" src="https://github.com/user-attachments/assets/8a6293b7-2576-48db-a65b-a992edb9eb03">


Pagina: [Persoonlijke gegevens inzien of een correctie doorgeven](https://www.amsterdam.nl/veelgevraagd/persoonlijke-gegevens-inzien-of-een-correctie-doorgeven-2bf85#)

---

### Notities

`<strong>` element wordt gebruikt in een Unordered list en in een lopende zin.

**Comment by Rozerinay** [comment]
## Den Haag

---

### Screenshots / Varianten

**Publieke website**

<img width="400" alt="image" src="https://github.com/user-attachments/assets/d7fdb153-1c84-4583-8918-eb65020de3e9">

<img width="400" alt="image" src="https://github.com/user-attachments/assets/cd234c12-ef3c-45ca-a653-de1583283f4c">

Pagina: [Gemeentebegroting 2025: Den Haag is financieel gezond](https://www.denhaag.nl/nl/nieuws/gemeentebegroting-2025-den-haag-is-financieel-gezond/)

---

### Notities
`<strong>` element wordt gebruikt voor een heel alinea. Er wordt aangeraden om CSS te gebruiken om tekst vetgedrukt te maken zonder semantische betekenis.

**Reply by julezrulez** [reply]
Door hele alinea's (of zinnen) te markeren met het strong-element voldoe je niet aan WCAG, succescriterium 1.3.1.

**Comment by Rozerinay** [comment]
## Rotterdam

---

### Screenshots / Varianten

**Publieke website**

<img width="550" alt="image" src="https://github.com/user-attachments/assets/baa09759-9eb9-4efe-837e-f2992cd1cff8">

Pagina: [Parkeervergunning bedrijven en onderwijsinstellingen aanvragen](https://arc.net/l/quote/ypkgwonx)

<img width="550" alt="image" src="https://github.com/user-attachments/assets/4d4eb08b-8002-4edf-b277-fc905b43a68e">

Pagina: [Grofvuil wegbrengen of laten ophalen](https://www.rotterdam.nl/grofvuil-wegbrengen-of-laten-ophalen)

<img width="550" alt="image" src="https://github.com/user-attachments/assets/dd4872f7-f15c-4a83-a477-ab3f407b5e7a">

Pagina: [Parkeervergunning bewoner aanvragen](https://www.rotterdam.nl/parkeervergunning-bewoner-aanvragen)

<img width="803" alt="image" src="https://github.com/user-attachments/assets/138bc073-4a0b-46cd-b4c8-381300605eca">

Pagina: [Zoekresultaten](https://www.rotterdam.nl/zoeken?zoekopdracht=aow)

---

### Notities
`<strong>` element wordt gebruikt om ontkennende woorden (‘geen’) en een imperatief (‘let op’) te benadrukken. Verder komt het voor in een Notification en bij de zoekresultaten om de zoekopdracht te benadrukken.

**Comment by Rozerinay** [comment]
## Westland

---

### Screenshots / Varianten

**Publieke website**

<img width="550" alt="image" src="https://github.com/user-attachments/assets/69adbc55-cc9c-4570-ab26-e1bde8abc087">

Pagina: [Grofvuil](https://www.gemeentewestland.nl/aanvragen-en-regelen/afval-en-huisvuil/grofvuil)

---


### Notities
`<strong>` element wordt gebruikt om woorden in een Unordered list te benadrukken.

**Comment by Rozerinay** [comment]
## Rijksoverheid

---

### Screenshots / Varianten

**Publieke website**

<img width="300" alt="image" src="https://github.com/user-attachments/assets/b256f29c-2ea0-4d9e-9bba-2cde32ae1ec6"> <br>
<img width="300" alt="image" src="https://github.com/user-attachments/assets/0609d595-3ccb-44d0-9d52-2a508cdd98a3"> 
Pagina: [AOW documenten](https://www.rijksoverheid.nl/onderwerpen/algemene-ouderdomswet-aow/documenten)

<img width="762" alt="image" src="https://github.com/user-attachments/assets/fc455690-8309-4448-8f08-17278acd2fe8">

Pagina: [Dick Schoof](https://www.rijksoverheid.nl/regering/bewindspersonen/dick-schoof#:~:text=Opleiding-,1975%2D1982,-Planologie%2C%20Katholieke%20Universiteit)

<img width="746" alt="image" src="https://github.com/user-attachments/assets/493901f5-df63-489a-9f09-3b6404415bea">

Pagina: [Contact](https://www.rijksoverheid.nl/contact)

---


### Notities
`<strong>` element wordt gebruikt bij het sorteren en in een Unordered list.

**Reply by julezrulez** [reply]
Visueel zijn de vetgedrukte teksten hier koppen. Door deze teksten met het strong-element te markeren voldoe je niet aan WCAG, succescriterium 1.3.1. Je voldoet hier door kop-elementen (h2, h3, ... h6) te gebruiken.

**Comment by Rozerinay** [comment]
## Omgevingsloket

---

### Screenshots / Varianten

**Publieke website**

<img width="1141" alt="image" src="https://github.com/user-attachments/assets/b80fcffa-916f-4200-a9f5-acffa916ee10">

Pagina: [Inloggen](https://omgevingswet.overheid.nl/inloggen?session=c6f261f2-96d5-4dc0-b882-b8c67c178e65)

<img width="617" alt="image" src="https://github.com/user-attachments/assets/60de7ef7-002e-48ca-99a0-47772e481beb">

Pagina: [Regels op de kaart](https://omgevingswet.overheid.nl/orienteren/viewer/overzicht?regelsbeleid=regels&locatie=Wilhelmina%20van%20Pruisenweg%20104,%202595AN%20's-Gravenhage&session=c6f261f2-96d5-4dc0-b882-b8c67c178e65)

---

### Notities
`<strong>` element wordt gebruikt om de doelgroep bij het inlogmiddel aan te geven en bij een gekozen adres.

**Comment by Rozerinay** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Strong op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=strong)
